source/globalCommands.py

```diff
@@ -2028,6 +2028,24 @@ def script_activateConfigProfilesDialog(self, gesture):
 	script_activateConfigProfilesDialog.__doc__ = _("Shows the NVDA Configuration Profiles dialog")
 	script_activateConfigProfilesDialog.category=SCRCAT_CONFIG
 
+	def script_toggleConfigProfileTriggers(self,gesture):
+		if config.conf.profileTriggersEnabled:
+			config.conf.disableProfileTriggers()
+			# Translators: The message announced when temporarily disabling all configuration profile triggers.
+			state = _("Configuration profile triggers disabled")
+		else:
+			config.conf.enableProfileTriggers()
+			# Explicitly trigger profiles for the current application.
+			mod=api.getForegroundObject().appModule

```

Comment:
Missing space around the `=` sign.

Reply:
Are there conventions for this? I'm happy for this to be changed, but I belief there are more cases in the code where equals isn't surrounded by spaces.

Reply:
> Are there conventions for this?

No, but multiple ways of dealing with whitespace in a single function makes me sad. :)